README.md

```diff
@@ -323,12 +323,80 @@ Templates in React are written inside the JavaScript file using the [JSX languag
 
 # Interpolation
 
-> TODO
+### AngularJS
+Interpolation is the process of data-binding values on the AngularJS `scope` to values in the html. You can read more on the [official documentation](https://docs.angularjs.org/guide/interpolation):
+
+Let's say we have a value `heroImageUrl` on our scope that is defined as `superman.jpg`.
+
+The following HTML
+
+```html
+<p><img src="images/{{heroImageUrl}}"> is the <i>interpolated</i> image.</p>
```

Comment:
That's actually an anti-pattern. People shouldn't use interpolation in the `src` attribute. There's `ng-src` for that:
https://docs.angularjs.org/api/ng/directive/ngSrc